ssl/ssl_lib.c

```diff
@@ -4132,6 +4132,26 @@ EVP_PKEY *SSL_CTX_get0_privatekey(const SSL_CTX *ctx)
         return NULL;
 }
 
+int SSL_CTX_remove_cert_and_key(SSL_CTX *ctx)
```

Comment:
I think I'd prefer `SSL_CTX_remove_certificate_and_key` for the function name, but it's not a blocking point.

Reply:
This looks analogous to the existing `SSL_CTX_use_cert_and_key()`.